libraries/debug-certificate-manager/src/CertificateManager.ts

```diff
@@ -21,11 +21,16 @@ const ONE_DAY_IN_MILLISECONDS: number = 24 * 60 * 60 * 1000;
  * The set of names the certificate should be generated for, by default.
  * @public
  */
-export const DEFAULT_CERTIFICATE_SUBJECT_NAMES: ReadonlyArray<string> = [
-  'localhost',
-  'rushstack.localhost',
-  '127.0.0.1'
-];
+export const DEFAULT_CERTIFICATE_SUBJECT_NAMES: ReadonlyArray<string> = ['localhost'];
```

Comment:
Removing `rushstack.localhost`?

Reply:
The issue here mostly has to do with that adding it, due to the rules, forces cert regeneration, which is a headache for developers consuming it.